### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a-zorina**
обновить до https://www.notion.so/tonlabs/TON-Labs-Validator-with-SafeMultiSig-Setup-HOWTO-4f2fc4ab4b844c36b2c13e9078d80904

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,106 @@
...
+Let `N` be the total number of wallet custodians and `K` the number of minimal confirmations required to execute a wallet transaction.
+
+1. Read [TONOS-CLI documentation](https://docs.ton.dev/86757ecb2/v/0/p/94921e-running-tonos-cli-with-tails-os-and-working-with-multisignature-wallet) (*Deploying Multisignature Wallet to TON blockchain*) and generate seed phrases and public keys for `N - 1`  custodians.
+2. Generate wallet address and `Nth` custodian key:
+```
+    $ cd ./scripts/ 
+    $ ./msig_genaddr.sh
+```
+Script creates 2 files: `$(hostname -s).addr` and `msig.keys.json` in `~/ton-keys/` folder. 
+Put them next to `validator_msig.sh` script.
```
Comment (ar-tmp) [inline README.md]: Remove this - need to keep the keys where they are generated.

File: README.md [hunk 2]
```diff
@@ -1,82 +1,106 @@
...
-command to generate Multisig address. Take public key from this file as a **custodian key** when you will deploy Multisig.
 
-Put `msig.keys.json` file near `validator.sh` script.
+### 2. Run Validator script
 
-### 6. Run Validator
+Specify `<STAKE>` argument in tokens. This amount of tokens will be sent by wallet to Elector smart contract in every validation cycle.
 
-Define env variable `stake` equal to amount of validator stake in tokens. This amount of tokens will be sent by multisig to Elector smart contract in every validation cycle. Otherwise script will use a hardcoded value (see `stake_hardcoded` v  ar).
+Run the validator script with the validator name as first argument:
```
Comment (ar-tmp) [inline README.md]: Change to `Run the validator script:`

File: README.md (hunk 1 above)
Comment (a-zorina) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (a-zorina) [reply README.md]: done

**Review by ar-tmp — APPROVED** [review]

**Pull request merged**